### PR TITLE
gobindata remove pin

### PR DESCRIPTION
**Pull request by mjudeikis**
```release-note
NONE
```

Stop pinning go-bindata to commit

related to: https://github.com/openshift/release/pull/4727 
old topic: https://github.com/openshift/openshift-azure/pull/862 
fix: https://github.com/openshift/openshift-azure/issues/1639 
fix: https://github.com/openshift/openshift-azure/issues/1045

### CODE REVIEW

**Comment by kwoodson** [comment]
I don't have any insight how this affects us other than gobin has advanced.  Is there any issue with us unpinning?  Are there any risks?

**Comment by mjudeikis** [comment]
| I don't have any insight how this affects us other than gobin has advanced. Is there any issue with us unpinning? Are there any risks?

Would be good to document these things better next time why we did this in the first place :)

**Comment by mjudeikis** [comment]
related: https://github.com/openshift/release/pull/4736

File: hack/verify/validate-generated.sh [hunk 1]
```diff
@@ -19,10 +19,6 @@ mkdir -p "${T}/src/github.com/openshift/openshift-azure"
 
 cp -a "${DIR}/../.." "${T}/src/github.com/openshift/openshift-azure"
 
-(GOPATH="${T}" go get -d "github.com/go-bindata/go-bindata/go-bindata" && cd "${T}/src/github.com/go-bindata/go-bindata" && git checkout -q 41975c0)
-(cd "${T}/src/github.com/openshift/openshift-azure" && GOPATH="${T}" go generate ./...)
```
Comment (m1kola) [inline hack/verify/validate-generated.sh]: We need to keep this line it seems. Otherwise the script will not be failing. I'll fix it

**Pull request merged**